docs/learn/prompts.md

```diff
@@ -71,12 +97,16 @@ print(prompt)
 # > Recommend a book cheaper than $12.34
 ```
 
+In this example, we use the `.2f` format specifier to limit the price to two decimal places.
+
 We also provide additional specifiers we've found useful in our own prompt engineering:
 
 - `list`: formats an input list as a newline-separated string
 - `lists`: formats an input list of lists as newline-separated strings separated by double newlines
 
-```python
+Here's an example demonstrating these custom specifiers:
+
+```python hl_lines="7 10 19-23"
```

Comment:
when using the function based approach, since the output is `list[BaseMessageParam]` we can always run `print(prompt(...)[0].content)` so it's pretty